### ISSUE TITLE
MAML checkpoint request for mini-Imagenet

### ISSUE THREAD

**Issue by brando90** [learnables/learn2learn#280]
Is it possible to request some checkpoints for mini-imagenet?

**Comment by seba-1511** [comment]
Hello @brando90,

We're slowly adding pretrained weights for some backbones (you can track #250 for details) but likely won't have them for MAML. The reason is simple: unlike other methods, we would have to host a different set of weights for every setting (1-shot, 5-shot, 10-shot, etc) and for every dataset, and that's just too much overhead.

**Comment by brando90** [comment]
Thanks @seba-1511 !

**Comment by brando90** [comment]
@seba-1511 Hi Seb!

I'm not requesting (right now) weights for every possible setting. I am requesting the MAML weights used for the sanity checks/unit tests you did for l2l. I remember you mentioned somewhere you got to reproduce with the l2l framework the 63% that MAML got from Chelsea Finn's original paper. Do you have **those weights** available to share?

**Comment by seba-1511** [comment]
So I dug into some more recent MAML runs on mini-ImageNet. With a bit of tuning, I can get on average 67%+ accuracy. My code is tied to a larger private codebase but very similar to the open-source one. Here are the hyper-parameter config I use:

```python

    @dataclasses.dataclass
    class FSLArgs:

        model: str = 'cnn4'
        dataset: str = 'mini-imagenet'
        iterations: int = 60000
        ways: int = 5
        shots: int = 5
        lr: float = 3e-4
        batch_size: int = 8
        eval_freq: int = 10
        eval_lr: float = 0.1
        eval_steps: int = 1
        first_order: bool = False
        cnn_channels: int = 64
        cuda: bool = True
        seed: int = 1234
        use_wandb: bool = True
        save_weights: bool = False
```

some of which are overridden with the following config file:

```yaml
eval_lr:
  desc: null
  value: 0.1
eval_steps:
  desc: null
  value: 3
first_order:
  desc: null
  value: 0
fsl.batch_size:
  desc: null
  value: 8
fsl.cnn_channels:
  desc: null
  value: 64
fsl.cuda:
  desc: null
  value: true
fsl.dataset:
  desc: null
  value: mini-imagenet
fsl.eval_freq:
  desc: null
  value: 10
fsl.eval_lr:
  desc: null
  value: 0.1
fsl.eval_steps:
  desc: null
  value: 3
fsl.first_order:
  desc: null
  value: false
fsl.iterations:
  desc: null
  value: 60000
fsl.lr:
  desc: null
  value: 0.0003
fsl.model:
  desc: null
  value: cnn4
fsl.save_weights:
  desc: null
  value: true
fsl.seed:
  desc: null
  value: 1
fsl.shots:
  desc: null
  value: 5
fsl.use_wandb:
  desc: null
  value: true
fsl.ways:
  desc: null
  value: 5
seed:
  desc: null
  value: 1
```

Sorry for not providing more details, as mentioned on slack I'll have limited time this summer.
